src/i18n/Localizable.xcstrings

```diff
@@ -21919,6 +22032,7 @@
       }
     },
     "Show_Password" : {
+      "comment" : "is this one used as gerund or noun?",
```

Comment:
Verb

Reply:
deleted comment and used gerund, same reason for "Restart"